docs/source/quicktour.rst

```diff
@@ -324,13 +338,13 @@ Post-processing
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 We might want our tokenizer to automatically add special tokens, like :obj:`"[CLS]"` or
-:obj:`"[SEP]"`. To do this, we use a post-processor. :class:`~tokenizers.TemplateProcessing` is the
+:obj:`"[SEP]"`. To do this, we use a post-processor. :entity:`TemplateProcessing` is the
 most commonly used, you just have so specify a template for the processing of single sentences and
```

Comment:
I apologize for pointing out silly typo.
Shouldn't this be `you just have to` instead of `you just have so`?

Reply:
No, that's great thanks! I really appreciate it!